lib/dialects/sqlite/query-interface.js

```diff
@@ -152,7 +153,7 @@ function addConstraint(tableName, options) {
   const describeCreateTableSql = this.QueryGenerator.describeCreateTableQuery(tableName);
   let createTableSql;
 
-  return this.sequelize.query(describeCreateTableSql, options)
+  return this.sequelize.query(describeCreateTableSql, { type: QueryTypes.SELECT, raw: true })
```

Comment:
We should pass other `options`, probably `{ type: QueryTypes.SELECT, raw: true, ...options }`

Reply:
That seemed to make things fail (although I never did try to figure out why). `options` was always something like
```javascript
{ type: 'unique', fields: [ 'email' ] }
```

I believe `options` is meant to be the options to create the constraint, not just general query options. Are there some specific properties I could pick out? I wasn't really sure of what the possibilities were.